packages/customize-widgets/src/components/error-boundary/index.js

```diff
@@ -11,12 +11,7 @@ import { doAction } from '@wordpress/hooks';
 function CopyButton( { text, children } ) {
 	const ref = useCopyToClipboard( text );
 	return (
-		<Button
-			// TODO: Switch to `true` (40px size) if possible
-			__next40pxDefaultSize={ false }
-			variant="secondary"
-			ref={ ref }
-		>
+		<Button size="compact" variant="secondary" ref={ ref }>
```

Comment:
- Active an non-blocks based theme
- Artificially `throw` somewhere in the package (
- Visit customizer, then widgets

![Screenshot 2024-10-01 at 22 09 32](https://github.com/user-attachments/assets/0dfa73cf-aec1-4ccd-9097-035cbc992cdc)

Note: this button inherits some [very generic style overrides](https://github.com/WordPress/gutenberg/blob/619d0e4bf2b8f4366283a1adff0c261006af929a/packages/customize-widgets/src/style.scss#L18), although this code is so legacy that we can probably keep it as-is without worrying too much.

Reply:
I'm okay with the style overrides FWIW, but I think this is a perfect fit for the `size="compact"`, given the size of the neighbor buttons that are likely to not change in the future.

Reply:
Makes sense — so, just to make sure, no changes needed for this instance since `size="compact"` is already being applied?

Reply:
That's correct 👍 

And whether we want to play with the style overrides in this PR in order to make the button height match better, I'll leave this to your judgment.

Reply:
I'd leave it since it's a legacy part of the codebase, and this error screen should (hopefully) not be seen often by users 🤞